package.json

```diff
@@ -36,21 +36,21 @@
     "express": "^4.17.1",
     "file-system": "^2.2.2",
     "full-icu": "^1.3.1",
-    "handlebars": "^4.7.6",
+    "handlebars": "^4.7.7",
     "jambo": "^1.12.1",
     "jest": "^25.5.2",
     "libphonenumber-js": "^1.9.6",
     "loader-utils": "^2.0.0",
     "locale-currency": "0.0.2",
     "lodash.clonedeep": "^4.5.0",
-    "postcss": "^8.2.1",
+    "postcss": "^8.4.14",
     "puppeteer": "^10.2.0",
     "serve": "^11.3.2",
     "simple-git": "^2.24.0",
-    "testcafe": "^1.18.4",
+    "testcafe": "^1.18.6",
```

Comment:
could we use 1.19 for testcafe?

Reply:
it looks like v1.19 requires node >= 14. since we still want to support node 12 in the Theme, I'll leave this as is